providers/aws/aws_provider.py

```diff
@@ -118,36 +117,61 @@ def validate_credentials(validate_session):
 
 
 def provider_set_session(session_input):
+
+    # global variables that are going to be shared accross the project
     global aws_session
     global original_session
+    global audited_regions
+    global audited_partition
+    global audited_account
```

Comment:
Why not include all of these values into the session object? I think these are session related and it would be easier to access them. What do you think?

Reply:
Task created to change it in the future.